proxy/envoy/resources.go

```diff
@@ -84,6 +84,13 @@ const (
 	// MONGOProxyFilter is the name of the MONGO Proxy network filter.
 	MONGOProxyFilter = "mongo_proxy"
 
+	// REDISProxyFilter is the name of the REDIS Proxy network filter.
```

Comment:
style nit: This should probably be `RedixProxyfilter` unless REDIS is an [initialism](https://github.com/golang/go/wiki/CodeReviewComments#initialisms). Similar comment applies elsewhere w.r.t. REDIS and MONGO.